homeassistant/components/climate/const.py

```diff
@@ -84,6 +84,7 @@
 CURRENT_HVAC_DRY = "drying"
 CURRENT_HVAC_IDLE = "idle"
 CURRENT_HVAC_FAN = "fan"
+CURRENT_HVAC_AUTO = "auto"
```

Comment:
This is a new "auto" current state for when we can't get a more precise action out of the HVAC device.

Reply:
This PR is changing our entity model. Before we can review this PR there needs to be approval in a discussion in our architecture repository.

https://github.com/home-assistant/architecture/discussions

https://developers.home-assistant.io/docs/core/entity#changing-the-entity-model